terminus-ui/confirmation/src/confirmation.directive.ts

```diff
@@ -104,6 +128,23 @@ export class TsConfirmationDirective implements OnDestroy, OnInit {
   }
   private _explanationText: string | undefined;
 
+  /**
+   * Define position of the modal
+   */
+  @Input()
+  public set modalPosition(value: TsConfirmationModalPositionTypes) {
```

Comment:
I know you inherited the `modal` verbiage here (thanks, me!), but I'd like to throw out the idea of adjusting from `modal` to `overlay` for everything.

Possible reasons for change:

a) The library we are using under the covers (CDK) uses the term `overlay` so this would align us better with our tools.
b) Modal is a loaded term at Terminus. Prior to this feature, the term modal was only visible internally but now it will be part of the component API.
c) It should probably happen in this PR because once this feature is released, this change would be a breaking change.

---

We should address all instances of `modal` including filenames, types, methods etc. It seems or CSS is already using `overlay`.

```
confirmation-modal.component -> confirmation-overlay.component
TsConfirmationModalPositionTypes -> TsConfirmationOverlayPositionTypes
modalPosition -> overlayPosition
allowedModalPositionTypes -> allowedOverlayPositionTypes
modalPos___ -> overlayPos
```